.gitignore

```diff
@@ -13,6 +13,7 @@
 /confdefs.h
 /conftest*
 /libtool
+/libgap.pc
```

Comment:
BTW, why `libgap.pc` and not `gap.pc`? (I don't want to bikeshed, and I am fine with keeping it as-is! I am merely curious and wondering if there is a deeper rationale for these names that you are aware of. Looking at the `pkgconfig` dir of one of my machines, I see `gap.pc`, `mpfr.pc`, `zlib.pc` yet `libcurl.pc` and `libzmq.pc` -- so it seems pretty much random?

Reply:
To me, `libgap.pc` is pretty much what's needed to include `libgap` in the project. But `gap.pc` is a bit unclear.